### DISCUSSION TITLE
Obtaining hindered rotor property correction for a single top (e.g. methyl)

### DISCUSSION THREAD

**Discussion by cassguff** [ReactionMechanismGenerator/RMG-Py#2587] [Q&A]
I am new to trying out RMG so maybe this is a dumb question. I'm wondering if it has the capability to help with what I am trying to do (or if anyone has any other ideas of a different tool I could use). Basically, I am doing opt/freq calculations for molecules in Gaussian and using the RRHO model to get the properties. I have my own code for that, that's easy. Now I want to obtain the contribution to the properties (eg ideal gas heat capacity) of just the methyl group top rotation using the 1DHR model. Is there a way for me to take a dihedral scan performed in Gaussian and use RMG to get the ideal gas heat capacity contribution of a hindered rotor? I want to add that to the calculation I've done for the rest of the molecule as a correction.

**Comment by JacksonBurns** [comment]
Not a 'dumb' question at all! @alongd do you have any input on this?

**Comment by cassguff** [comment]
I have been playing with the toluene_HinderedRotor example to get a feel
for using this. The output file from Arkane gives the total thermodynamic
properties. Is it possible to have the output file also contain the
individual contributions to each of these properties? I.E. the torsional,
rotational, translational, vibrational contributions to each of the
properties separately? I assume somewhere in the code the contributions are
calculated separately and just added together and then that result is what
goes to the output file. Would it be possible for me to get the values
before they are all added together?
